src/winetricks.appdata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2017 Daniel Rusek <mail@asciiwolf.com> -->
+<component type="desktop">
+  <id>winetricks.desktop</id>
+  <project_license>GPL-2.1</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Winetricks</name>
+  <summary>Work around problems and install applications under Wine</summary>
+  <description>
+    <p>
+    Winetricks is an easy way to work around problems in Wine.
+    </p>
+    <p>
+    It has a menu of supported games/apps for which it can do all the workarounds automatically.
+    It also lets you install missing DLLs or tweak various Wine settings individually.
```

Comment:
I'd prefer if the order of these were flipped. Winetricks is mostly used for DLLs/settings, not apps/games.